src/controllers/postsRoute/getFollowingPosts.js

```diff
@@ -0,0 +1,26 @@
+const { db } = require("../../database/database");
+
+exports.getFollowingPosts = async function (request, response) {
+  try {
+    const { username } = request.loggedInUser;
+
+    const user = await db.users.findOne({ username: username });
+
+    const { following } = user;
+
+    if (!Array.isArray(following)) {
+      return response.status(400).json({ error: "Invalid input data" });
```

Comment:
Blir det inte lite märkligt att skicka 400 till frontenden om vi får ett oväntat svar från våran databas när det vi skickar till databasen är username som vi hämtat från kakan som vi satt dit? Vore det inte bättre med 500? Det är ju inte användaren som skrivit in fel.

Reply:
kikar på de